modules/exceptionHandler.js

```diff
@@ -0,0 +1,95 @@
+        } 
+        else if(error.toString().includes(`Only absolute URLs are supported`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}: ${error}`, `Current provider URL is not supported by network provider....`);
+        } 
+        else if(error.toString().includes(`getaddrinfo ENOTFOUND`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}: ${error}`, `Current provider URL is not supported by network provider....`);
+        } 
+        else if(error.toString().includes(`Only HTTP(S) protocols are supported`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}: ${error}`, `Current provider URL is not supported by network provider....`);
+        } 
+        else if(error.toString().includes(`Cannot read property '0' of undefined`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}: ${error}`, `Current provider URL is not supported by network provider....`);
+        } 
+        else {
+            Logger.error(`Error occurred while ${commandType}:\n${error}`);
+        }
+
+    }
+
+    static transactionException(commandType, error) {
+
+        switch(commandType) {
+            case 'transfer':
+                commandType = 'transferring tez';
+                break;
+            case 'activateAccount':
+                commandType = 'activating account';
+                break;
+            default:
+                commandType = 'fetching balance';
+        }
+
+        if(error.toString().includes(`checksum`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}:${error}`, `Account doesn't  exists or not revealed on the network.... To list down all accounts run 'tezster list-accounts'`);
+        } 
+        else if(error.toString().includes(`empty_implicit_contract`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}:${error}`, `Account is not yet revealed on the blockchain. You can reveal the account by sending some tezos to the account.`);
+        } 
+        else if(error.toString().includes(`connect ECONNREFUSED`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}: ${error}`, `Error occurred while establishing the connection with node provider....`);
+        }
+        else if(error.toString().includes(`Only absolute URLs are supported`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}: ${error}`, `Current provider URL is not supported by network provider....`);
+        } 
+        else if(error.toString().includes(`getaddrinfo ENOTFOUND`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}: ${error}`, `Current provider URL is not supported by network provider....`);
+        } 
+        else if(error.toString().includes(`Invalid`)) {
+            Helper.errorLogHandler(`Error occurred while ${commandType}: ${error}`, `Current provider URL is not supported by network provider....`);
+        } 
```

Comment:
you can create constant for each error message, 
this way it'll be reusable and also easy to modify.